nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/ui/registry-clients/registry-client-table/registry-client-table.component.html

```diff
@@ -0,0 +1,119 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="relative h-full border">
+    <div class="registry-client-table listing-table absolute inset-0 overflow-y-auto">
+        <table mat-table [dataSource]="dataSource" matSort matSortDisableClear>
```

Comment:
probably should add an initial sort:
```suggestion
        <table mat-table [dataSource]="dataSource" matSort matSortDisableClear matSortActive="name" matSortDirection="asc">
```

but, of course that would introduce ExpressionChangedAfterItHasBeenCheckedError if we continue to use the `even` helper in the row def on line 116 below.

So we should probably implement the sort function here.